libs/ledger/include/ledger/protocols/main_chain_rpc_service.hpp

```diff
@@ -91,6 +91,8 @@ class MainChainRpcService : public muddle::rpc::Server,
   MainChainRpcService(MainChainRpcService &&)      = delete;
   ~MainChainRpcService() override                  = default;
 
+  void Start();
```

Comment:
Maybe I am going mad but where is this called from? Does it also make sense to have a stop the drops the subscription on the floor too?

Reply:
Urgh, forgot to include that. thanks.